rando_modules/modify_entrances.py

```diff
@@ -295,27 +296,26 @@ def get_bowsercastle_bossrush(world_graph: dict):
     """
     # Sets up the following connection:
     # hos_20 (2) <-> kpa_53 (0) (Riding Star Ship Scene <-> Fake Peach Hallway)
-    all_entrance_modifications = []
+    all_new_edges = []
+    all_edges_to_remove = []
+    entrance_modifications = []
 
-    world_graph, hos_entrance_modifications = adjust(
-        world_graph,
-        new_edges=edges_hos_add,
-        edges_to_remove=edges_hos_remove
-    )
-    world_graph, kpa_entrance_modifications = adjust(
+    all_new_edges.extend(edges_hos_add)
```

Comment:
Do we need to deepcopy the edges? Having PTSD from the star hunt saga where not doing it caused issues. Not sure if it applies here.

Reply:
We do not need to deepcopy here. That is only needed if we either manipulate an edge afterwards (like in get_starhunt) or use the same edge multiple times (like in _set_rowf_out_of_logic etc.), both of which we do not do here.